crypto_analytics/collection/data_source/kraken.py

```diff
@@ -1,17 +1,21 @@
-import pandas as  pd
-import json
-import krakenex
+import pandas as pd
+import numpy as np
+import time
+import requests
 from typing import Dict
 
 from crypto_analytics.collection.data_source import OHLCVDataSource
 from crypto_analytics.types import Interval
 
 class KrakenOHLCV(OHLCVDataSource):
     columns = ['time', 'open', 'high', 'low', 'close', 'vwap', 'volume', 'count']
+    # TODO: define appropriate dtypes
+    dtypes = {'time': np.int64, 'open': object, 'high': object, 'low': object, 'close': object, 'vwap': object, 'volume': object, 'count': np.int64 }
 
-    def __init__(self, interval: Interval, pair: str, since: int = None):
+    def __init__(self, interval: Interval, pair: str, rows: int, last_time: int = None):
```

Comment:
I'm trying to move towards unifying common data source params. Eventually we should have some sort of `SymbolPair` type but I think `rows` and `last_time` might be common data source parameters that'll help me with writing a collection script.